BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.kt

```diff
@@ -396,7 +398,11 @@ open class BrickDataManager : Serializable {
      * Method called to release any related resources.
      */
     open fun onDestroyView() {
-        recyclerView?.adapter = null
+        try {
+            recyclerView?.adapter = null
+        } catch (e: IllegalStateException) {
+            Log.w(TAG, "Catching RecyclerView not registered exception")
+        }
         brickRecyclerAdapter = null
         recyclerView = null
```

Comment:
it looks like those two lines should be enough:
brickRecyclerAdapter = null
recyclerView = null